test/sanity/code-smell/no-smart-quotes.py

```diff
@@ -15,25 +15,27 @@ def main():
         'test/integration/targets/template/files/encoding_1252_utf-8.expected',
         'test/integration/targets/template/files/encoding_1252_windows-1252.expected',
         'test/integration/targets/template/templates/encoding_1252.j2',
+        'docs/docsite/_build/',
     ])
 
     for path in sys.argv[1:] or sys.stdin.read().splitlines():
-        if path in skip:
-            continue
-
-        with open(path, 'rb') as path_fd:
-            for line, text in enumerate(path_fd.readlines()):
-                try:
-                    text = text.decode('utf-8')
-                except UnicodeDecodeError as ex:
-                    print('%s:%d:%d: UnicodeDecodeError: %s' % (path, line + 1, ex.start + 1, ex))
-                    continue
-
-                match = re.search(u'([‘’“”])', text)
-
-                if match:
-                    print('%s:%d:%d: use ASCII quotes `\'` and `"` instead of Unicode quotes' % (
-                        path, line + 1, match.start(1) + 1))
+        for skip_path in skip:
+            if path.startswith(skip_path):
```

Comment:
For consistency path pruning should be done the same as in the boilerplate test:

https://github.com/ansible/ansible/blob/60a24bbdaae3a17bf20613810f347bfb8283f306/test/sanity/code-smell/boilerplate.py#L13-L14

https://github.com/ansible/ansible/blob/60a24bbdaae3a17bf20613810f347bfb8283f306/test/sanity/code-smell/boilerplate.py#L41-L42

Reply:
Done.